api/v1/submissions/SubmissionHandler.inc.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+/**
+ * @file api/v1/submission/SubmissionHandler.inc.php
+ *
+ * Copyright (c) 2014-2016 Simon Fraser University Library
+ * Copyright (c) 2003-2016 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @class SubmissionHandler
+ * @ingroup api_v1_submission
+ *
+ * @brief Handle API requests for submission operations.
+ *
+ */
+
+import('lib.pkp.classes.handler.APIHandler');
+
+class SubmissionHandler extends APIHandler {
+
+	/**
+	 * The unique endpoint string for this handler
+	 *
+	 * @param string
+	 */
+	protected $_handlerPath = 'submissions';
```

Comment:
Cheers. I reckon this is something that's fallen out of sync with your own work as you've pushed forward on the public API. I'll get these changes in place, but you'll probably need to manually merge your new code in after this makes it into master.